doc/api/http.md

```diff
@@ -1539,6 +1539,8 @@ convenience method. The only difference between this method and
 automatically. Note that response data must be consumed in the callback
 for reasons stated in [`http.ClientRequest`][] section.
 
+Note: The `options` properties are copied via `util._extend()` which doesn't include the prototype.
```

Comment:
I'd like to avoid scare quotes in our docs if possible. Maybe this?:

> *Note*: Properties inherited from the `options` object prototype chain are ignored.

Might it be better as part of the explanation of `options` and not a special note at the bottom?

>  * `options` {Object | string} Accepts the same `options` as
>   [`http.request()`][], with the `method` always set to `GET`.
>  Properties that are inherited from the prototype are ignored.

Also: Should this note be added to all relevant functions and not just `.get()`?

@nodejs/documentation